src/components/SplitFlow.vue

```diff
@@ -323,7 +406,48 @@ export default {
     },
     contextUpdate: function (payload) {
       EventBus.emit("contextUpdate", payload);
-    }
+
+      const categoryValues = [];
+      const { label, type, datasetId, resource } = payload;
+      let filePath = '';
+      if (label) categoryValues.push(label);
+      if (type) categoryValues.push(type);
+      if (datasetId) categoryValues.push('(' + datasetId + ')');
+      if (resource) {
+        filePath = resource.share_link;
+      }
+
+      // GA Tagging
+      // Event tracking for map sidebar gallery click
+      Tagging.sendEvent({
```

Comment:
This should be kept in the actionClick function if possible. There are gallery items that do not trigger this event later on due to the lack of a context card.

Reply:
This is updated now. The tracking event is moved inside actionClick.
Thanks for highlighting.